scrapper.py

```diff
@@ -1,6 +1,20 @@
 """
 Scrapper implementation
 """
+from datetime import datetime
+import json
+from pathlib import Path
+import re
+from random import randint
+import shutil
+from time import sleep
+from bs4 import BeautifulSoup
+import requests
+from constants import ASSETS_PATH
+from constants import DOMAIN_NAME
+from constants import CRAWLER_CONFIG_PATH
+from constants import HEADERS
```

Comment:
you can safely unify importing several objects from the same module: from constants import a, b, c...